src/shared/components/case-editor/case-edit-page/case-edit-page.component.ts

```diff
@@ -134,56 +134,38 @@ export class CaseEditPageComponent implements OnInit, AfterViewChecked {
   }
 
   // Adding validation message to show it as Error Summary
-  public generateErrorMessage(fields: CaseField[], container?: AbstractControl): void {
-    const group: AbstractControl = container || this.editForm.controls['data'];
+  public generateErrorMessage(fields: CaseField[]): void {
```

Comment:
Everything in this method is 100% wrong. This undoes all the fixes @RiteshHMCTS put in place!